src/Raven.Studio/typescript/common/queryCompleter.ts

```diff
@@ -453,7 +453,10 @@ class queryCompleter {
         callback(null,  keywords.map(keyword  => {
             const word = <autoCompleteWordList>keyword;
             word.caption = _.trim(keyword.value, "'");
-            keyword.value += " ";
+            if (keyword.value.indexOf(" ") >= 0){
```

Comment:
String has includes method :)

Reply:
I expected it to have `Contains`... thanks, I'll update the code.